src/aqua/installation-spell/src/aqua/status.aqua

```diff
@@ -0,0 +1,51 @@
+import Spell from "@fluencelabs/spell/spell_service.aqua"
+import Peer from "@fluencelabs/aqua-lib/builtin.aqua"
+import StatusJson from "./json.aqua"
+import Status, State from "./types.aqua"
+
+const STATUS_KEY = "__installation_spell_status__"
+
+const STATUS_UNDEFINED = "STATUS_UNDEFINED"
+const INSTALLATION_IN_PROGRESS = "INSTALLATION_IN_PROGRESS"
+const INSTALLATION_FAILED = "INSTALLATION_FAILED"
+const INSTALLATION_SUCCESSFUL = "INSTALLATION_SUCCESSFUL"
+
+func get_status() -> Status:
+    Spell "worker-spell"
+    statuses <- Spell.list_get_strings(STATUS_KEY)
+    status: ?Status
+
+    if statuses.strings.length == 0:
+        status <<- Status(state = STATUS_UNDEFINED, message = "Installation has not started yet", timestamp = 0)
+    if statuses.success == false:
+        status <<- Status(state = STATUS_UNDEFINED,  message = statuses.error, timestamp = 0)
```

Comment:
Why is it undefined here? Sounds like `STATUS_LOAD_ERROR` to me or something like that.